streaming/base/converters/dataframe_to_mds.py

```diff
@@ -237,16 +237,16 @@ def write_mds(iterator: Iterable):
                 for sample in records:
                     try:
                         mds_writer.write(sample)
-                    except:
-                        raise RuntimeError(f'failed to write sample: {sample}')
+                    except Exception as ex:
+                        raise RuntimeError(f'failed to write sample: {sample}') from ex
                         count += 1
```

Comment:
This is kinda useless anyway because this code is unreachable...

Reply:
Thanks. I thought I deleted it. It must slipped from the ~300 conversations! Can you remove counting from the function in this PR as well?

Reply:
@Skylion007 have you found any better ways to surface the workers' debug output to SparkUI?

Reply:
The raise from is definitely a better way.